bot/cogs/sync/syncers.py

```diff
@@ -217,11 +60,14 @@ async def sync(self, guild: Guild, ctx: t.Optional[Context] = None) -> None:
 
             # Don't show response text because it's probably some really long HTML.
             results = f"status {e.status}\n```{e.response_json or 'See log output for details'}```"
-            content = f":x: {mention}Synchronisation of {self.name}s failed: {results}"
+            content = f":x: Synchronisation of {self.name}s failed: {results}"
         else:
-            results = ", ".join(f"{name} `{total}`" for name, total in totals.items())
+            diff_dict = diff._asdict()  # Ugly method for transforming the NamedTuple into a dict
```

Comment:
The comment doesn't add much here, I think we should just remove it.

Reply:
I recall it was requested by @lemonsaurus 🤷

Reply:
Don't think there's anything ugly about it with since it is a properly documented method with a clear name https://docs.python.org/3.8/library/collections.html#collections.somenamedtuple._asdict

Reply:
I agree with Numerlor. Can't really remember talking about this with you, but I forget a lot of stuff like this so maybe. Maybe it's because of the leading underscore? It is a bit ugly but I also think it's crystal clear.

Reply:
https://github.com/python-discord/bot/pull/711#discussion_r383047221

Anyway, I'll remove it